### PR TITLE
Dev 1498-Use the nested multi choice selector for industries and functions in e.g. Postion Requirements Page rather than separate dropdowns

### PR DESCRIPTION
**Pull request by EnriqueCherryfish**

### CODE REVIEW

**Comment by EnriqueCherryfish** [comment]
I forgot to remove the WIP commit :( :S

File: packages/app-web/src/components/Form/__tests__/TieredMultiChoiceSelect.test.tsx [hunk 1]
```diff
@@ -47,228 +43,92 @@ const createWrapper = (props: Partial<Props> = {}) => {
...
+      industrySubdomain: [v('industry two')],
+      industryCategory: [v('new industry category')]
+    }
+
+    const {select, onChange} = createWrapper({values})
+    select.simulate('change', {
+      industryDomain: v('new industry'),
+      industrySubdomain: v('new industry'),
+      industryCategory: {
+        label: 'different new industry category',
```
Comment (mattcherryfish) [inline packages/app-web/src/components/Form/__tests__/TieredMultiChoiceSelect.test.tsx]: ?

Comment (mattcherryfish) [reply packages/app-web/src/components/Form/__tests__/TieredMultiChoiceSelect.test.tsx]: Ah okay, I get it now. The original label "A different label, but the same value" Is a bit clearer, I think

Comment (EnriqueCherryfish) [reply packages/app-web/src/components/Form/__tests__/TieredMultiChoiceSelect.test.tsx]: i will change to the original

**Pull request merged**